docs/_templates/layout.html

```diff
@@ -118,19 +118,22 @@
     <!-- top 48px for navbar height -->
     <div class="navbar-cet">
         <div style="padding: 0 16px; display: flex; height: inherit; align-items: center; justify-content: space-between;">
-            <span style="display:flex; gap: 32px; align-items: center;">
-                <a href="https://www.czimaginginstitute.org/">
-                  <img src="{{ pathto('_static/img/czi-imaging-institute.png', 1) }}" style="height: 45px; width: 45px;" />
-                </a>
-                <span style="display:flex; gap: 16px;">
-                    <span class="navbar-cet-link">
-                      <a class="navbar-cet-anchor" href="index.html">CryoET Data Portal | Documentation</a>
-                  </span>
-                </span>
+            <span style="display:flex; gap: 8px; align-items: center;">
+              <a href="index.html" style="color: #fff">CryoET Data Portal</a>
+              <img alt="beta" src="{{ pathto('_static/img/beta.svg', 1) }}">
+              <div style="width: 1px; height: 16px; background: #fff; margin: 0 16px"></div>
```

Comment:
@codemonkey800 i think the divider's left & right margins are too large & too bright, can we:

- reduce left/right margin to 4px
- change "background" color to #999